dns/bind/Makefile

```diff
@@ -1,6 +1,5 @@
 PLUGIN_NAME=		bind
-PLUGIN_VERSION=		1.33
-PLUGIN_REVISION=	1
+PLUGIN_VERSION=		1.34
```

Comment:
I think I did this morning, but you can reuse this version, just needs small amendment in pkg-descr